app/extensions/brave/locales/en-US/app.properties

```diff
@@ -43,6 +43,7 @@ reloadTab=Reload tab
 reloadButton.title=Reload page
 homeButton.title=Open Homepage
 stopButton.title=Stop loading
+noScriptButton.title=Allow scripts
```

Comment:
There might be better ones like "Scripts Blocked", "Scripts are blocked", "NoScripts enabled", "click here to allow scripts", etc...